### PR TITLE
Added logic to handle improper video resource access attempts.

### PR DESCRIPTION
**Pull request by eschlange**
* Added logic to controller to handle invalid video ids.
* Added redirect and error message logic to video show action.
* Added specs for new logic.

[#109744480]

### CODE REVIEW

**Comment by eschlange** [comment]
@ericcf Let me know if you are overburdened today. I can merge small PRs like this myself if so.

File: app/controllers/videos_controller.rb [hunk 1]
```diff
@@ -7,6 +9,8 @@ def index
   end
 
   def show
+    redirect_to "/", flash: { error: "Please view assigned videos "\
```
Comment (ericcf) [inline app/controllers/videos_controller.rb]: Is there a reason for the non-standard usage of `redirect_to` here?

e.g.
```ruby
redirect_to root_url, alert: "Please view ..."
```

Comment (eschlange) [inline app/controllers/videos_controller.rb]: No, I was just copying usage from existing code within the app. I'll update it to conform to its standard usage.

Comment (ericcf) [inline app/controllers/videos_controller.rb]: Ok, in that case I'd opt for consistency over fixing this specific usage.

Comment (eschlange) [inline app/controllers/videos_controller.rb]: OK, in the future I'll use this format.

**Comment by ericcf** [comment]
:rotating_light:

**Pull request merged**